less/navigation.less

```diff
@@ -1 +1,78 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+header {
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items:center;
+    margin:0px auto;
+    padding:0;
+    width:100%;
+    background-color:@porcelain;
+  
+    h1 {
+        width:100%;
+        .flex-centered();
+        padding:1.25%;
+        color:@ebony-clay;
+        font-size: 2rem;
+        margin:15px;
+    }
+  
+    nav {
+        display:flex;
+        color:@river-bend;
+        width:45%;
+        justify-content:space-between;
+        padding-right:3%;
+        margin-bottom: 3%;
+        
+        a {
+            text-decoration:none;
+            color: black;
+            font-size:1.6rem;
+        }
+    }
+  }
+  
+  @media @phone{
```

Comment:
not cool. you have to nest your media query inside of the element that requires it